Dockerfile

```diff
@@ -402,27 +402,25 @@ ARG CGO_CFLAGS="${CFLAGS}"
 ARG CGO_CXXFLAGS="${CXXFLAGS}"
 ARG CGO_LDFLAGS="${LDFLAGS}"
 
-WORKDIR /home/builder/go/src
+WORKDIR /home/builder/sdk-go/src
 RUN ./make.bash --no-clean
 
 # Build the standard library with and without PIE. Target binaries
 # should use PIE, but any host binaries generated during the build
 # might not.
-WORKDIR /home/builder/go
+WORKDIR /home/builder/sdk-go
+ENV GOOS="${GOOS}" \
+  GOROOT="/home/builder/sdk-go" \
+  GOPATH="/home/builder/gopath" \
```

Comment:
Does building `go` with these values set have any consequences for the final binary - do they override what would be the default values?